dbus/dbus_test.go

```diff
@@ -80,9 +77,3 @@ func TestMain(m *testing.M) {
 
 	m.Run()
 }
-
-func TestGetDBusAPI(t *testing.T) {
-	api, err := GetDBusAPI()
-	assert.NoError(t, err)
-	assert.NotNil(t, api)
```

Comment:
I guess this test still applies, no?
Except for the `NoError` part?

Reply:
This cannot return anything but one value now, so the value of testing it is questionable. In any case, as mentioned, this part is going to change in the next task, so I'll probably write a new test as part of that instead.